Sources/Support/POSIX/Stringify.cpp

```diff
@@ -262,6 +262,14 @@ char const *Stringify::Errno(int error) {
 
 char const *Stringify::Ptrace(int code) {
   switch (code) {
+#if defined(OS_DARWIN)
```

Comment:
Put these at the end, and do `#elif defined(OS_DARWIN) || defined(OS_FREEBSD)`.

These values work for FreeBSD too so we might as well define them here.